apps/api/src/user/model-schemas/user-api-key/user-api-key.schema.ts

```diff
@@ -0,0 +1,28 @@
+import { sql } from "drizzle-orm";
+import { boolean, index, pgTable, timestamp, unique, uuid, varchar } from "drizzle-orm/pg-core";
+
+import { Users } from "@src/user/model-schemas";
+
+export const UserApiKeys = pgTable(
+  "user_api_keys",
+  {
+    id: uuid("id")
+      .primaryKey()
+      .notNull()
+      .default(sql`uuid_generate_v4()`),
+    userId: uuid("user_id")
+      .references(() => Users.id, { onDelete: "cascade" })
+      .notNull(),
+    apiKey: varchar("api_key").notNull(),
```

Comment:
A separate index declaration is not the only way. Unique index can be added on a field itself
```suggestion
    apiKey: varchar("api_key").notNull().unique(),
```